src/samples/thmviewer/thmviewer.vcxproj

```diff
@@ -66,4 +66,4 @@
   </ItemGroup>
 
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
You should have removed line 50.

Also, why did you modernize the .wixproj when it isn't even getting built?

Reply:
Still working on that part.